seedfarmer/__init__.py

```diff
@@ -113,5 +119,11 @@ def OPS_ROOT(self) -> str:
             self._load_config_data()
         return str(self._OPS_ROOT)
 
+    @property
+    def PROJECT_POLICY_PATH(self) -> str:
+        if self._PROJECT_POLICY_PATH is None:
```

Comment:
No it will not, once the load function has been called, this parameter is initialized to the default value (the policy path packaged in the distro).  This is correct and should not be changed.

Reply:
But agreed it will be improved when moved to a class